MSALiOSB2C/ViewController.swift

```diff
@@ -105,8 +93,9 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
             
             application.acquireToken(forScopes: kScopes) { (result, error) in
                     if  error == nil {
```

Comment:
please check for the valid here result, instead of an error object. 
for this being a swift project, even more recommended take would be to use guard instead of error == nil.
i.e./
guard let validResult = result else { // handle error here } 
// continue authentication with validResult.
This will remove need for using "result?.XXX"

Reply:
Good hint, fixed everywhere.